app/serializers/activitypub/actor_serializer.rb

```diff
@@ -147,6 +147,50 @@ def name
     end
   end
 
+  class FeaturedTagSerializer < ActivityPub::Serializer
+    include RoutingHelper
+
+    attributes :type, :href, :name, :featured
+
+    def type
+      'Hashtag'
+    end
+
+    def href
+      short_account_tag_url(Account.find(object.account_id), object.name)
+    end
+
+    def name
+      "##{object.name}"
+    end
+
+    def featured
```

Comment:
@nightpool Wait, so `featured` used to be the link to the pinned posts collection. Is it okay to re-use it for a boolean value with a different semantic meaning? I have a feeling "maybe not"

Reply:
yeah pretty sure this is not going to be okay for any software doing proper json-ld. maybe 'promoted' or 'endorsed' or 'highlighted' or 'recommended' or some other similar word. i'm personally for 'recommended' as in 'i recommend you browse this hashtag of mine' or 'i recommend you check out this cool profile', but no strong feelings.